src/services/query.js

```diff
@@ -621,6 +621,22 @@ ngeo.Query.prototype.doGetFeatureRequests_ = function(
         return;
       }
 
+      // add dimensions values
+      var dimensions = items[0].source.dimensions;
+      if (dimensions) {
+        for (var key in dimensions) {
+          // get the value from the global dimensions
+          var value = this.dimensions[key];
+          if (value === undefined) {
+            // get the value from the layer default value
+            value = dimensions[key];
+          }
+          if (value !== undefined) {
+            url = goog.uri.utils.setParam(url, key, value);
```

Comment:
We try to don't use goog anymore to be able to build the project without google library. 
In the master branch almost all the goog (Except goog.[asserts, requires, provide, module]) will be removes, in this case there a method in openlayers that can be used.